programs/Makefile

```diff
@@ -27,7 +27,7 @@ else
 ALIGN_LOOP =
 endif
 
-CPPFLAGS+= -I$(ZSTDDIR) -I$(ZSTDDIR)/common -I$(ZSTDDIR)/dictBuilder
+CPPFLAGS+= -I$(ZSTDDIR) -I$(ZSTDDIR)/common -I$(ZSTDDIR)/dictBuilder -I./deprecated
```

Comment:
Is it necessary there ?
`ZBUFF_*` is never used from `zstd` nor any of its variants.